cert/index.md

```diff
@@ -0,0 +1,103 @@
+# gNSI cert Service Protobuf Definition
+**Contributors**: hines@google.com, morrowc@google.com, tmadejski@google.com
+**Last Updated**: 2022-07-12
+
+## Background
+
+The cert service definition provides the API to be used for installing,
+rotating, and testing PKI primatives used on network systems. The Rotate
+and Install are bidirectional streaming rpcs which permit mutating
+Certificates, Root Certificate Bundles, Certificate Revocation Lists.
+For either an Install or Rotate stream it is possible to mutate one or
+more of the elements, and to send a Finalize message once the in-flight
+change has been verified to be operational. Failure to send the Finalize
+message will result in the candidate element being discarded and the
+original element being used instead.
+
+## Motivation
+
+Management of the PKI elements for a network system should have
+a clear and direct method for installation and update.
+
+### CertificateManagement.Install
+
+CertificateManagemetn.Install will permit installation, and
+verification of function, of any of the PKI elements. The normal
+use-case would be to:
+
+* send an Certificate and Key to a network system as an
+InstallCertificateRequest.
+* verify that the services which will use the certificate continues
+to operate normally.
+* send a FinalizeRequest to finish the installation process.
+
+### CertificateManagement.Rotate
+
+CertificateManagement.Rotate will permit rotation, and
+verification of function, of any of the PKI elements.
+The normal use-case would be to:
+
+* send an CertificateBundle to a network system as a
+RotateCertificateRequest.
+* verify that the services which will use the new certificate bundle
+continue to operate normally.
+* send a FinalizeRequest to finish the rotateion process.
+
+## User Experiences
```

Comment:
Suggest "User Journey"

Reply:
This is (the header) just copy/pasted from the healthz example, I don't have a preference, but I expect consistency matters across the docs of this type.